themes/default/content/docs/using-pulumi/adopting-pulumi/converters.md

```diff
@@ -13,38 +13,39 @@ aliases:
 - /docs/converters/
 ---
 
-Pulumi converters allow you to convert ARM, CloudFormation, Kubernetes CustomResources, Kubernetes YAML, and Terraform to Pulumi.
-
-<ul class="">
-    <li class="pt-4">
-        <a class="" href="/arm2pulumi">
-        ARM to Pulumi:
-        </a>
-        This conversion tool will do the magic of translating your ARM templates into modern code using Pulumi.
-    </li>
-    <li class="pt-4">
-        <a class="" href="/cf2pulumi">
-        CloudFormation to Pulumi:
-        </a>
-        This conversion tool will do the magic of translating your CloudFormation templates into TypeScript/JavaScript, Python, Go and C# using Pulumi.
-    </li>
-    <li class="pt-4">
-        <a class="" href="/blog/introducing-crd2pulumi">
-        Kubernetes CustomResources to Pulumi:
-        </a>
-        CustomResources in Kubernetes allow users to extend the API with their types. These types are defined using CustomResourceDefinitions (CRDs), which include an OpenAPI schema.
-        Our new crd2pulumi tool takes the pain out of managing CustomResources by generating types in the Pulumi-supported language of your choice!
-    </li>
-    <li class="pt-4">
-        <a class="" href="/kube2pulumi">
-        Kubernetes YAML to Pulumi:
-        </a>
-        This conversion tool will do the magic of translating your YAML into modern code using Pulumi.
-    </li>
-    <li class="pt-4">
-        <a class="" href="/tf2pulumi">
-        Terraform to Pulumi:
-        </a>
-        This conversion tool will do the magic of translating your HCL into modern code using Pulumi.
-    </li>
-</ul>
+`pulumi convert` is a subcommand within the Pulumi Command Line Interface (CLI) you can use to convert between different source Infrastructure as Code (IaC) languages or formats into all of Pulumi’s [supported languages](/docs/languages-sdks/), such as TypeScript, JavaScript, Go, Python, C#, or Java.
+
+For the detailed usage of this command and options, refer to the [pulumi convert CLI documentation.](https://www.pulumi.com/docs/cli/commands/pulumi_convert/)
+
+### Supported source languages
```

Comment:
This isn't really accurate because only Terraform and YAML are sources for the `pulumi convert` command. We also have legacy standalone conversion utilities like `arm2pulumi`, `cf2pulumi`, etc. We either need to change the scope of this page to be "conversion" in the general sense and include everything, or a `pulumi convert` usage guide and narrow the list of utilities to only those that are proper `pulumi convert` plugins.

It seems like the former is closer to the author's intent, so we should explain the "old vs. new" situation we have right now. I think the user cares a lot less about standalone executables versus a plugin model than they do "I have CloudFormation today. What are my options?"